src/picongpu/include/simulation_defines/param/ionizationEnergies.param

```diff
@@ -63,15 +63,36 @@ namespace AU
      * @see libPMacc/include/math/ConstVector.hpp
      * for finding ionization energies,
      * @see http://physics.nist.gov/PhysRefData/ASD/ionEnergy.html
+     *
+     * Reference: Kramida, A., Ralchenko, Yu., Reader, J., and NIST ASD Team (2014)
+     *            NIST Atomic Spectra Database (ver. 5.2), [Online]
+     *            Available: http://physics.nist.gov/asd [2017, February 8]
+     *            National Institute of Standards and Technology, Gaithersburg, MD
      */
 
     /* ionization energy for ground state hydrogen in atomic units */
-    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY_HYDROGEN,
+    PMACC_CONST_VECTOR(float_X, 1, IONIZATION_ENERGY_HYDROGEN,
         13.60569 * UNITCONV_eV_to_AU
```

Comment:
Hmm ... so do we change this to `13.598434005136` now as found [here](http://physics.nist.gov/cgi-bin/ASD/ie.pl)?